src/popup/router/components/BalanceInfo.vue

```diff
@@ -11,8 +11,7 @@
         <span class="approx-sign">~</span>
         <li data-cy="currency-dropdown" class="dropdown-container" :class="dropdown ? 'show' : ''">
           <ae-button data-cy="toggle-currency-dropdown" @click="dropdown = !dropdown">
-            {{ balanceCurrency }}
-            <span class="currency">{{ currentCurrency }}</span>
+            {{ formatCurrency(balanceCurrency) }}
```

Comment:
It would be tricky to keep this behavior using `Intl.NumberFormat`, so I propose to drop that.